src/js/mictest.js

```diff
@@ -128,8 +128,10 @@ MicTest.prototype = {
   testInputVolume: function(buffer, channel) {
     var data = buffer.getChannelData(channel);
     var sum = 0;
+    var s = 0;
```

Comment:
Can you rename it to something meaningful? Maybe replace "sample" with "i" and "s" with "sample"?